packages/python/plotly/_plotly_utils/basevalidators.py

```diff
@@ -1708,16 +1710,17 @@ def description(self):
     (e.g. 270 is converted to -90).
         """.format(
             plotly_name=self.plotly_name,
-            array_ok=", or a list, numpy array or other iterable thereof"
-            if self.array_ok
-            else "",
+            array_ok=(
+                ", or a list, numpy array or other iterable thereof"
+                if self.array_ok
+                else ""
+            ),
```

Comment:
🤔 different black version?

Reply:
Oh interesting! Yeah not sure why that's happening but it looks fine?

Reply:
Yeah I would not worry too much about this, I just wonder why that's the case 😇